dipu/torch_dipu/csrc_dipu/vendor/droplet/CMakeLists.txt

```
@@ -1,23 +1,29 @@
 cmake_minimum_required(VERSION 3.5)
 
-option(DIST "whether to use PCCL to implement communicator" OFF)
-
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 set(DROPLET_INSTALL "/usr/local/tangrt")
 include(cmake/FindDropletCompiler.cmake)
 
-if (DIST)
-    message(STATUS "use PCCL to implement communicator")
-    find_package(PCCL REQUIRED)
-    message(STATUS "PCCL_LIBRARY: " ${PCCL_LIBRARY})
-    message(STATUS "PCCL_INCLUDE_DIR: " ${PCCL_INCLUDE_DIR})
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_PCCL" PARENT_SCOPE)
-    set(DIPU_VENDOR_LIB ${DIPU_VENDOR_LIB} ${PCCL_LIBRARY})
-    set(VENDOR_INCLUDE_DIRS ${VENDOR_INCLUDE_DIRS} ${PCCL_INCLUDE_DIR})
-
+set(USE_PCCL "AUTO" CACHE STRING "Used to force load and compile PCCL. AUTO: auto load and compile PCCL; 1(TRUE or true): Force load and compile PCCL; 0(FALSE or false): Forbidden load PCCL")
+
+find_package(PCCL)
+if (USE_PCCL OR USE_PCCL STREQUAL "AUTO")
+    if (USE_PCCL AND  NOT PCCL_FOUND)
+        message(STATUS "Cannot find PCCL to implement communicator, not support distributed on droplet!")
+    endif()
+    if(PCCL_FOUND)
+        message(STATUS "Use PCCL to implement communicator")
+        message(STATUS "PCCL_LIBRARY: " ${PCCL_LIBRARY})
+        message(STATUS "PCCL_INCLUDE_DIR: " ${PCCL_INCLUDE_DIR})
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_PCCL" PARENT_SCOPE)
+        set(DIPU_VENDOR_LIB ${DIPU_VENDOR_LIB} ${PCCL_LIBRARY})
+        set(VENDOR_INCLUDE_DIRS ${VENDOR_INCLUDE_DIRS} ${PCCL_INCLUDE_DIR})
+    elseif(USE_PCCL STREQUAL "AUTO")
+        message(STATUS "Cannot auto load PCCL to implement communicator, not support distributed on droplet!")
+    endif()
```

这里简化吧，不判断 auto 了，错误输入 fallback 到 auto 也合理
```suggestion
    else()
        message(STATUS "PCCL not found, i.e. not supporting distributed on droplet")
    endif()
```